tests/api/test_reference_qc.py

```
@@ -931,6 +918,303 @@ def test_calculate_sex_chrs_metrics(self):
+        qc = ReferenceQC(
+            sample_sig=sample_sig_nonref,
+            reference_sig=self.reference_sig,
+            amplicon_sig=None,
+            enable_logging=False
+        )
+        
+        nonref_stats = qc.nonref_consume_from_vars(vars=vars_signatures, vars_order=vars_order)
+        
+        # Expected:
+        # non-var consumes all k-mers
+        expected_stats = {
```

Ruff

Local variable `expected_stats` is assigned to but never used

Remove assignment to unused variable `expected_stats`

(F841)